action.yml

```diff
@@ -1,9 +1,7 @@
 ---
 name: setup-beam
 description: >
-  Set up a BEAM-based CI environment, with Erlang/OTP as a base followed, optionally, by other
-  elements, like Elixir, Gleam, `rebar3` or `hex`, on top of that. Check the [README](https://github.com/erlef/setup-beam)
-  for updated information.
+  Set up an Erlang/OTP based CI environment with support for languages and tooling such as Elixir, Gleam, rebar3, and `hex`.
```

Comment:
I'd specifically (recently) changed to have "BEAM-based", because you told me searching for BEAM yielded no results in the Marketplace. 😕

Reply:
Yeah, I changed it to this because I was trying to cram all the things in, do you have a better suggestion?

Reply:
```
Set up a BEAM-based CI environment, supporting languages and tools such as Erlang/OTP, Elixir, Gleam, `rebar3`, and `hex`.
```

😄

Reply:
That'll work!